src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -250,6 +253,10 @@ nuget list -verbose -allversions</value>
   <data name="ListCommandVerboseListDescription" xml:space="preserve">
     <value>Displays a detailed list of information for each package.</value>
   </data>
+  <data name="Log_RestoreNoCacheInformation" xml:space="preserve">
+    <value>NoCache is deprecated in favor of the appropriately named NoHttpCache.</value>
```

Comment:
Can we check on wording of this? I'm not sure "appropriately named NoHttpCache" makes sense without explaining what NoCache does. Consider a few alternatives:
 
```suggestion
    <value>NoCache is deprecated. Please use NoHttpCache instead.</value>
```

```suggestion
    <value>NoCache is deprecated and has been renamed to NoHttpCache. Please use NoHttpCache instead.</value>
```